lib/surrogate_key_logging/sidekiq.rb

```diff
@@ -34,6 +33,7 @@ def job_hash_context(job_hash)
           end
           hash
         end
+        alias_method :job_hash_context__before_surrogate_key_logging, :job_hash_context
```

Comment:
Just curious what moving it below does? Any additional insight as to why you made this change?

Reply:
That has to go above, not below. It saves a link to the original method, so that it can be used in the override.

Reply:
In fact, I'm pretty sure putting it below will cause infinite recursion. If upgrading sidekiq caused an undefined method there, then you'll need to go look at the sideline source to see what they replaced that method with.

Reply:
Looks like they broke this entire sidekiq integration with https://github.com/sidekiq/sidekiq/commit/01686c5e25dd83ab89c6d85b36feec26e203e81b#diff-4b21ac0af44dd61654e41fe70e857785cf96a82c222ada536c3d812be0101452R105. The sidekiq integration will have to be done completely different if using a version that includes that commit.

Reply:
The discussion here https://github.com/sidekiq/sidekiq/discussions/5343 may help with an idea on how to do a new implementation.

Reply:
Nice find @yelvert ! Thanks for sharing.